core/src/apps/bitcoin/sign_tx/approvers.py

```diff
@@ -94,7 +95,10 @@ async def approve_tx(self) -> None:
 
         # fee > (coin.maxfee per byte * tx size)
         if fee > fee_threshold:
-            if fee > 10 * fee_threshold and not device.unsafe_prompts_allowed():
+            safety_checks_prompt = (
+                device.safety_check_level() == SafetyCheckLevel.Prompt
+            )
+            if fee > 10 * fee_threshold and not safety_checks_prompt:
```

Comment:
Since we are using `== Prompt` everywhere maybe we could introduce `device.safety_check_is_prompt()` (better wording maybe?) and then use that everywhere.

Reply:
yes please